hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoDeleteSuite.java

```diff
@@ -109,7 +111,8 @@ private HapiSpec deletedAccountCannotBePayer() {
                                 .logged());
     }
 
-    private HapiSpec canQueryForRecordsWithDeletedPayers() {
+    @HapiTest
+    HapiSpec canQueryForRecordsWithDeletedPayers() {
```

Comment:
Why did we expand the access modifier for this method (and the one above)l do we actually need to?

Reply:
Accidentally removed. Thanks for catching it. Updated.